tests/testthat/test-utils-html.R

```diff
@@ -135,7 +135,7 @@ test_that('formatQualifiersTable handles no qualifiers', {
 }')
   qualsDf <- repgen:::readQualifiers(inQualifiers,NULL)
   qualList <- repgen:::formatQualifiersTable(qualsDf)
-  expect_true(is.null(qualList))
+  expect_true(nrow(qualList)==0)
```

Comment:
had to change this test to check for a dataframe with 0 rows instead of null since dataframes can't be tested for null http://stackoverflow.com/questions/28556658/why-does-an-empty-dataframe-fail-an-is-null-test